devel/CImg/Portfile

```diff
@@ -3,27 +3,27 @@
 PortSystem          1.0
 
 name                CImg
-version             3.2.4
+version             3.3.0
 categories          devel
 platforms           any
 maintainers         yandex.com:bstj openmaintainer
 supported_archs     noarch
-license             CeCILL-C CeCILL
+license             {CeCILL-C CeCILL}
 
 description         C++ Template Image Processing Toolkit
 
 long_description    Open source, C++ toolkit for image processing
 
-homepage            http://cimg.eu \
+homepage            https://cimg.eu \
                     https://github.com/GreycLab/CImg
 master_sites        http://cimg.eu/files/
```

Comment:
HTTPS here too plz

Reply:
I think I tried to use it, and it failed to fetch. Will check again.

Reply:
No, it works fine. I will change it to `https` now.

Reply:
@herbygillot Fixed and rebased to master.